src/NServiceBus.RavenDB.AcceptanceTests/NServiceBus.RavenDB.AcceptanceTests.csproj

```diff
@@ -13,7 +13,7 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
     <PackageReference Include="NServiceBus" Version="7.2.2" />
     <PackageReference Include="NServiceBus.Gateway.AcceptanceTests.Sources" Version="3.1.0" />
-    <PackageReference Include="NServiceBus.AcceptanceTests.Sources" Version="7.2.1" />
+    <PackageReference Include="NServiceBus.AcceptanceTests.Sources" Version="7.2.2" />
```

Comment:
@bording would we consider removing the dependency on NServiceBus here or would we end up with a too old version?